src_new/util.js

```diff
@@ -324,11 +324,14 @@ exports.forEachGeneratedKey = function(adapterID, adUnits, adapterConfig, impres
 	}
 };
 
-exports.resizeWindow = function(theDocument, width, height){
+exports.resizeWindow = function(theDocument, width, height,divId){
 	/* istanbul ignore else */
 	if(height && width){
 		try{
 			var fr = theDocument.defaultView.frameElement;
+			if(divId){
+			 	 fr = document.getElementById(divId).querySelector('iframe')
+			}
```

Comment:
https://github.com/prebid/Prebid.js/commit/5b4e3828111dd3b9a6c5965fdcd67800c0bd838e
Refer above commit, we also need to resize the div

Reply:
@pm-shashank-jain  looks like you have missed this coment

Reply:
@pm-harshad-mane i did those changes

Reply:
Cool, done.